modelzoo/utils.py

```diff
@@ -0,0 +1,66 @@
+import typing
+import base64
+import io
+import os
+from w3lib.url import parse_data_uri
+import mimetypes
+from PIL import Image
+from pathlib import Path
+from _protos.services_pb2 import (
+    TextGenerationRequest,
+    ImageSegmentationRequest,
+    VisionClassificationRequest,
+    ModelUUIDRequest,
+    ModelUUIDResponse
+)
+
+class UnauthorizedException(Exception):
+    pass
+
+ImgLike = typing.NewType('ImgLike', typing.Union[type(Image), str])
+
+def img_file_to_uri(filepath: str) -> str:
+    """
+        Reads an JPEG image from a file and converts it to a data URI.
```

Comment:
Not just JPEG. It should be any PIL compatible format. Also, the dotstring format should follow https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html